pkg/yamltemplate/starlark_fragment.go

```diff
@@ -17,6 +17,7 @@ const (
 	starlarkFragmentType = "yamlfragment"
 )
 
+// StarlarkFragment is an adapter for yamlmeta.Node values in Starlark code
```

Comment:
Great to see the godocs comment added here! It will definitely help with understanding this code.